web/Views/Admin/LandlordList.cshtml

```diff
@@ -6,18 +6,32 @@
 
 <h2>Landlord List</h2>
 
-<div class="mb-3">
-    @using (Html.BeginForm("LandlordList", "Admin", FormMethod.Get))
-    {
-        <select class="form-select" id="approvalStatus" name="approvalStatus">
-            <option disabled selected>Filter by approval status</option>
-            <option value="">Display all landlords</option>
-            <option value="Unapproved">Display unapproved landlords</option>
-            <option value="Approved">Display approved landlords</option>
-        </select>
-        <button type="submit" class="btn custom-btn-primary">Display results</button>
-    }
-</div>
+<a class="btn btn-primary" asp-controller="Landlord" asp-action="RegisterGet" asp-route-createUnassigned="true">Create Unassigned Landlord</a> <br/>
+
+<p></p>
+@using (Html.BeginForm("LandlordList", "Admin", FormMethod.Get))
+{
+    <div class="row g-3">
+        <div class="col-auto d-grid">
+            <select class="form-select" id="Approved" name="Approved">

```

Comment:
Can you use `asp-for` or whatever it's called instead of specifying `name`? That way if we rename the properties on LandlordListModel these should be updated as well by Rider.

Reply:
I talked to Oskar about these forms, and he suggested rewriting them as Html.DropDownListFor() - the property name is now defined by model => model.IsApproved, so I think that should work as you suggest anyway.

Reply:
Yeah that sounds fine. For the record the reason we haven't been using the helpers for selects everywhere is that as far as I can tell you can't do placeholder (i.e. a default value that you can't see in the dropdown) values with `selected disabled`, whereas you can with the tag helper things.

But that's not relevant here since `all` is an option that we want to be selectable, not just a placeholder.